theforeman.org/yaml/triggers/github_pr.yaml

```diff
@@ -39,7 +39,7 @@
           <<: *ghprb_defaults
           status-url: https://ci.theforeman.org/blue/organizations/jenkins/$JOB_NAME/detail/$JOB_NAME/$BUILD_ID/pipeline
           white-list-target-branches:
-            - rpm/3\.[0-8]+
+            - rpm/3\.8
```

Comment:
Wouldn't a more correct fix have been to only drop `+`? I'm fine with it now, until we need to revive 3.7 ;)

Reply:
Let's just not revive it then? ;)

Reply:
I would have also accepted "He's dead, Jim"